filter/ArticleMedraXmlFilter.inc.php

```diff
@@ -331,6 +405,21 @@ function createContentItemNode($doc, $issue, $article, $galley, $objectLocalePre
 			}
 
 		}
+		//Adding Citation List (unstructured) on the content
+		$citationDao = DAORegistry::getDAO('CitationDAO'); /* @var $citationDao CitationDAO */
+		$parsedCitations = $citationDao->getBySubmissionId($article->getId())->toArray();
+		$citationListNode = $doc->createElementNS($deployment->getNamespace(), 'cl:CitationList');
```

Comment:
Shall the element exist even if there is no citation?

Reply:
Put the verification of not empty array or non null array

Reply:
Please let me know if the new implementation is correct? Regards.